spec/service/car_cleaner_service_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../../service/car_cleaner_service'
+require 'timecop'
+
+describe CarCleanerService do
+  before { Timecop.freeze(Time.local(2021, 4, 20, 8, 0, 0)) }
+  let(:car_cleaner_service) { CarCleanerService.new Time.now }
+  
```

Comment:
my suggestion is to try to be consistent in the way you are calling the methods, for example here you are using `CarCleanerService.new Time.now ` without parentheses but then you also use `Car.new("CJ29TST", "2021-04-21")` with parentheses. This is a general comment, I noticed through the code that both ways are used.

Reply:
I sometimes don't use paranthesis if there is only one argument, but indeed it can get confusing. I'll use paranthesis everywhere.